client/rest-high-level/src/main/java/org/elasticsearch/client/RequestConverters.java

```diff
@@ -191,6 +192,16 @@ static Request putMapping(PutMappingRequest putMappingRequest) throws IOExceptio
         return request;
     }
 
+    static Request getMappings(GetMappingsRequest getMappingsRequest) throws IOException {
+        Request request = new Request(HttpGet.METHOD_NAME, endpoint(getMappingsRequest.indices(),
+            "_mapping", getMappingsRequest.types()));
+
+        Params parameters = new Params(request);
+        parameters.withMasterTimeout(getMappingsRequest.masterNodeTimeout());
```

Comment:
I don't see this being set in the corresponding REST action, would you mind opening another PR to fix that? It needs to be added to the SPEC too.